City-of-Helsinki/drupal-helfi-rekry
# UHF-7010: Translations

## What was done

Added missing translations for job listing.

## How to install

1. Set up REKRY instance
2. Checkout this branch: `git checkout UHF-7010_translations`
3. Import translations and config: `make shell`, `drush locale:check && drush locale:update` and `make drush cim -y`
4. Clear cache: `drush cr`

## How to test
See any job listing in all Finnish and Swedish and make sure that there are no original English strings visible.
